src/software/sensor_fusion/sensor_fusion.cpp

```diff
@@ -180,6 +180,40 @@ void SensorFusion::updateWorld(
     }
 }
 
+bool SensorFusion::shouldTrustRobotStatus(
+    const SSLProto::SSL_DetectionFrame &ssl_detection_frame,
+    const std::vector<BallDetection> &ball_detection)
```

Comment:
nit: both arguments seem to be unused. We should remove them if we actually dont need them.

Reply:
agreed.